General queries/WD AV Signature and Platform Version.txt

```diff
@@ -7,6 +7,8 @@
 let startDate = ago(7d);
 FileCreationEvents
 | where InitiatingProcessCommandLine has "MpSigStub.exe"
+//To exclude Engine Updates and non update events 
+| where InitiatingProcessParentFileName != "AM_Engine.exe" and InitiatingProcessParentFileName != "wuauclt.exe"
```

Comment:
Please use `!~` operator, which is case insensitive, instead `==` which is case sensitive.

Reply:
Created new commit with the suggested change.